src/components/speaker/SpeakerDetails.graphql

```diff
@@ -5,5 +5,8 @@ query SpeakerDetails($id: ID!) {
     avatar
     name
     biography
+    talks {
+      id
```

Comment:
This should be
```graphql
talks {
  id
  ...AgendaTalksListItem
}
```

This allows us to fetch all of the data in one query, instead of multiple (one query to fetch the speakers data, then an additional query for every talk that they have). We only need to pass `id` as a prop to `AgendaTalksListItem` though (which is what you've done) and Apollo will be smart about figuring out that it doesn't need to re-fetch that data.

Reply:
I have changed the query.